data/formats-data.js

```diff
@@ -618,7 +618,7 @@ exports.BattleFormatsData = {
 	},
 	machamp: {
 		randomBattleMoves: ["dynamicpunch","icepunch","stoneedge","bulletpunch","knockoff","substitute"],
-		randomDoubleBattleMoves: ["dynamicpunch","helpinghand","protect","icepunch","rockslide","bulletpunch","knockoff","wideguard"],
+		randomDoubleBattleMoves: ["dynamicpunch","protect","icepunch","stoneedge","rockslide","bulletpunch","knockoff","wideguard"],
```

Comment:
Yeah, thanks to No Guard, Stone Edge is used more than Rock Slide by 0.6 percentage points. It still gets rejected if Rock Slide is also present, of course.